labs/SizerBase/samples/SizerBase.Samples/SizerBase.Samples.csproj

```diff
@@ -13,10 +13,34 @@
     <RootNamespace>SizerBaseExperiment.Samples</RootNamespace>
```

Comment:
We don't need to modify the csproj as the files will be picked up automatically.

Reply:
changes made by removing the code added to the .csproj file